src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -80,12 +122,39 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
     }
   }
 
-  private onMutationError(errors, errorModalMessage?) {
+  private routeToArtworkPage() {
+    const artworkId = get(
+      this.props.order,
+      o => o.lineItems.edges[0].node.artwork.id
+    )
+    // Don't confirm whether or not you want to leave the page
+    this.props.route.onTransition = () => null
```

Comment:
Faffed with this a little bit, but this was the cleanest way I could find to ignore the transition hook behavior just in this case.

Reply:
Interesting... We have a cleanup method in OrderApp, but it's not necessarily available here... hmm... 

https://github.com/artsy/reaction/blob/master/src/Apps/Order/OrderApp.tsx#L46

Reply:
Yeah :/ if you think there's a better pattern to adopt let me know. In their example they delegate all handling to the children, but that doesn't seem much better than what we're doing here.

Reply:
Nope, this is definitely good enough. I don't think it's worth the effort.